### ISSUE TITLE
Camera not using full display/screen

### ISSUE THREAD

**Issue by SquirrelsMcGee** [signalapp/Signal-Android#11852]
I truly could not find a duplicate in the issue tracker.

- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Every time I open the camera to take a photo, the preview doesn't use the full display, and the button to take the picture is too high. 

This happens both when using the front and rear camera.

### Steps to reproduce
- Open the camera

**Actual result:** Large blank/black area below the camera preview, button appears too high
**Expected result:** Camera preview utilises entire display, button appears at the bottom, there is no black area

### Screenshots
![image](https://user-images.githubusercontent.com/8070535/146693558-5c7d8642-80ad-4d1f-afd4-8ad87fa86a98.png)

### Device info

**Device:** Oneplus 7 Pro
**Android version:** 11
**Signal version:** 5.28.5

### Link to debug log
https://debuglogs.org/9bfd268e91ace3c159a8d9e196ee7eee7bf71c5c8722873f36c0d363fae99c6f

**Comment by SquirrelsMcGee** [comment]
Appreciate the fast response, could I get a link to the discussion thread where this decision was made for context?

**Comment by greyson-signal** [comment]
There wasn't a discussion thread, we just made the decision. It used to be the case that full-screen image capture was in vogue, but sending images in many different sizes depending on your screen resolution led to an inconsistent experience, and most photo-taking apps moved to a standard resolution, so we did the same. I agree that the positioning of the button on tall displays could be better though, we'll look into it.
